esp/esdllib/README.md

```diff
@@ -290,6 +293,9 @@ flowchart TD
     needEsdl3--no-->preLogging[[PreLogging: \n Transform Log Request]]
 
     needEsdl3--yes-->esdlResponse[[EsdlResponse: \n Transform initial ESP response \n to final ESP response]]
+
+    esdlResponse-->send((Send\nResponse))
+
```

Comment:
This change doesn't make sense.

<img width="478" alt="image" src="https://user-images.githubusercontent.com/17124815/227215796-b8d1d263-21f9-4f01-a273-b959cc65102e.png">

Reply:
fixed